build-config.yaml

```diff
@@ -1,6 +1,6 @@
 # Options intended to be common for all developers.
 
-wasm-size-limit: 15.87 MiB
+wasm-size-limit: 15.85 MiB
```

Comment:
We probably shouldn't revert this line. I think it had to be bumped in other PRs as well.